pages/1.11/release-notes/1.11.2/index.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.11.2
+title: Release Notes for 1.11.2
+menuWeight: 0
+excerpt: Release notes for DC/OS 1.11.2
+---
+
+These are the release notes for DC/OS 1.11.2.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.2/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.2 includes the following:
+
+- Apache Mesos 1.5.x [change log](https://github.com/mesosphere/mesos/blob/27d91e1fe46f09b2c74f2dc4efe4f58ae59ae0a8/CHANGELOG).
+- Marathon 1.6.392 [change log](https://github.com/mesosphere/marathon/releases).
+- Metronome 0.4.2 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.2).
+
+
+# Issues Fixed in DC/OS 1.11.2
+
+- DCOS-14199 - Fixed an issue that prevented DC/OS from recovering when available disk space was low.
+- DCOS-21596 -  Added local user to LDAP group if local username matches LDAP username. [enterprise type="inline" size="small" /]
+- DCOS-22016 - Updated Prometheus documentation.
+- DCOS-22128 - Supported pods with volumes but no volume mounts. [enterprise type="inline" size="small" /]
+- DCOS-34465 - Updated documentation on using multiple methods to SSH to a master node. 
+- DCOS_OSS-2132 - Added a new journal reader function `Follow()` to track the `journald` files rotation.
+- DCOS_OSS-2360 - Ordered numeric characters for Metric names in dcos-metrics Prometheus endpoint.
```

Comment:
This wording does not explain the change (how I understand it).

Reply:
@philipnrmn can you reword this change, so I can document the actual change?

Reply:
I think I understood the issue now and provided an alternative in the engineering changelog documents linked here: https://github.com/mesosphere/dcos-docs-site/pull/991#issuecomment-389590581

Reply:
@Poltergeist reworded this change as:
DCOS_OSS-2360 - DC/OS Metrics: metric names are sanitized for better compatibility with Prometheus.

Reply:
Cool, thanks!